docpages/03_example_programs.md

```diff
@@ -1637,11 +1637,11 @@ The above is just a very simple example. You can also send embed messages. All y
 
 ### What is an eval command anyway?
 
-Many times people will ask: "how do i make a command like 'eval' in C++". For the uninitiated, an `eval` command is a command often found in interpreted languages such as Javascript and Python, which allows the developer to pass in raw interpreter statements which are then executed within the context of the running program, without any sandboxing. Eval commands are plain **evil**.
+Many times people will ask: "how do i make a command like 'eval' in C++". For the uninitiated, an `eval` command is a command often found in interpreted languages such as Javascript and Python, which allows the developer to pass in raw interpreter statements which are then executed within the context of the running program, without any sandboxing. Eval commands are literally dangerous if not used correctly.
 
 Needless to say, this is very dangerous. If you are asking how to do this, and want to put this into your bot, we trust that you have a very good reason to do so and have considered alternatives before resorting to this. The code below is for educational purposes only and makes several assumptions:
 
-1. This code will only operate on UNUX-like systems such as Linux
+1. This code will only operate on UNIX-like systems such as Linux (doesn't include Darwin, like Mac OS though)
```

Comment:
"Mac OS" should be "macOS"

Reply:
I dont see a difference between them. Also it's better to pronounce Mac OS and not macOS

Reply:
it is supposed to be stylized as "macOS" not"Mac OS".

Reply:
Dpp doesn't even focus on Mac, and neither do I. Anyways mac os is a shit, and I don't want to focus a lot of that simple spacing in their name.

Reply:
Dpp doesn't even focus on Mac, and neither do I. Anyways mac os is a shit, and I don't want to focus a lot of that simple spacing in their name.